vpr/src/route/route_common.cpp

```diff
@@ -86,7 +86,7 @@ static int num_linked_f_pointer_allocated = 0;
  *                                                                          */
 
 /******************** Subroutines local to route_common.c *******************/
-static t_trace_branch traceback_branch(int node, std::unordered_set<int>& main_branch_visited);
+static t_trace_branch traceback_branch(int node, int target_pin, std::unordered_set<int>& main_branch_visited);
```

Comment:
I'd use a more descriptive variable name:  target_net_pin_index would be better I think

Reply:
Good idea, while I'm at it, I also changed the parameter name in update_traceback, which is the routine a level above this.